src/app/containers/ArticleTimestamp/index.jsx

```diff
@@ -59,11 +64,27 @@ const ArticleTimestamp = ({
 
   const timeDifferenceMinutes = (lastPublished - firstPublished) / 1000 / 60;
 
+  const hasBeenUpdated = timeDifferenceMinutes > minutesTolerance;
+
+  const publishedAndUpdatedOnSameDay = isSameDay(firstPublished, lastPublished);
+
+  const publishedToday = isToday(firstPublished);
+
+  const publishedAndUpdatedToday =
+    publishedAndUpdatedOnSameDay && publishedToday;
+
+  const publishedAndUpdatedOnSameDayAndNotRelative =
+    publishedAndUpdatedOnSameDay && !isLastRelative(lastPublished);
+
+  const showLastUpdatedTimestamp =
+    hasBeenUpdated &&
+    (publishedAndUpdatedToday || !publishedAndUpdatedOnSameDayAndNotRelative);
+
```

Comment:
@jamesdonoh 

> I find the negated `!publishedAndUpdatedOnSameDayOverTenHoursAgo` a little hard to follow (maybe because of the negated `!updatedLessThanTenHoursAgo` 'within' it?). 

Understand why this double-negative feels a little mucky. Struggling to think of a better way to do it though. 

Within ``` const publishedAndUpdatedOnSameDayAndNotRelative =
    publishedAndUpdatedOnSameDay && !isLastRelative(lastPublished); ``` the !isLastRelative condition is important because this variable is determining that the content was published/updated on the same day, while we are no longer within the 'isLastRelative' time window.

Within ``` const showLastUpdatedTimestamp =
    hasBeenUpdated &&
    (publishedAndUpdatedToday || !publishedAndUpdatedOnSameDayAndNotRelative); ``` Human explanation: if it was publishedAndUpdatedToday, we want to show the second timestamp. If it wasn't published and updated today, then the only other circumstance where we would want to hide the second timestamp (and I guess this is where the confusion is - because it becomes easier to create the 'hidden' scenario rather than all possible 'show' scenarios) is if it was published and updated on the same day, and that day is outside of the 'relative' window (currently > 10 hrs ago).